jax/_src/lib/xla_bridge.py

```diff
@@ -297,9 +287,9 @@ def backends():
   with _backend_lock:
     if _backends:
       return _backends
-
-    if FLAGS.jax_platforms:
-      jax_platforms = FLAGS.jax_platforms.split(",")
+    default_priority = -1000
```

Comment:
You can move this back to where it originally was now (it's good to define variables as close to where you use them as possible)